lib/options.nix

```diff
@@ -403,5 +403,41 @@ rec {
         if cfg ? lazyLoad then lib.literalMD "`false` when lazy-loading is enabled." else true;
       example = false;
     };
+
+  /**
+    Create an option for an executable currently unpackaged in nixpkgs.
+    This option will throw an error if a value is not explicitly set by the end user
+  */
+  mkUnpackagedOption =
+    fullName: name:
```

Comment:
Q: would `pluginName` and `packageName` be better?

Reply:
Not sure, maybe `optionName` and `packageName`?